core/src/main/scala/cats/data/IndexedStateT.scala

```diff
@@ -387,11 +387,11 @@ private[data] sealed abstract class IndexedStateTContravariantMonoidal[F[_], S]
   implicit def F: ContravariantMonoidal[F]
   implicit def G: Applicative[F]
 
-  override def unit[A]: IndexedStateT[F, S, S, A]  =
-    IndexedStateT.applyF(G.pure((s: S) => F.unit[(S, A)]))
+  override def unit: IndexedStateT[F, S, S, Unit]  =
+    IndexedStateT.applyF(G.pure((s: S) => F.trivial[(S, Unit)]))
```

Comment:
It's reported that this instance is not tested. Shall add one?

Reply:
the ContravariantMonoidal instance of IndexedStateT requires that F to have both FlatMap and ContravriantMonoidal. If Phantom is the only one that satisfies that I don’t know if it’s worthwhile to have it.

Reply:
I incline to remove this instance, WDYT @stephen-lazaro ?

Reply:
Agreed, it seems at best awkward and unuseful.

Reply:
Agreed, it seems at best awkward and unuseful.